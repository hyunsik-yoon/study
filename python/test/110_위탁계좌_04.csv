거래일자,거래번호,거래구분,적요명,종목코드,종목명,수량,단가,거래금액,수수료,거래세,소득세,농특/주민세,세금합계,정산금액,미수변제/연체,기타대여/연체,유가잔고,예수금금잔,대용잔고,약정금액,대출(상환)금액,대출이자,대체금융회사,대체계좌번호,계좌명,받는분 표시내용,보내는분 표시내용,과표
2023-09-11,1,매수-장내,매수      ,A285130     ,얼바인,10,100,"1,000",100,0,,,0,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-12,1,매도-장내,매도      ,A285130     ,얼바인,2,200,400,40,44,,,44,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-21,2,매도-장내,매도      ,A285130     ,얼바인,6,300,"1,800",180,198,,,198,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-22,1,매수-장내,매수      ,A285130     ,얼바인,2,310,620,62,0,,,0,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-24,2,매도-장내,매도      ,A285130     ,얼바인,1,220,220,22,24,,,24,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
2023-09-26,2,매도-장내,매도      ,A285130     ,얼바인,3,250,750,75,82,,,82,,,,,,,,,,                                                                                ,                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ,                                                  
0000-01-01,,,,,이 테스트는 매도 두개에 매수가 여러개 존재하는 테스트,,,,,,,,,,,,,,,,,,,,,,,
