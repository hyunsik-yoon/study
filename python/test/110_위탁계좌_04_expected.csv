매수거래일자,매도거래일자,종목명,거래수량,총수수료,총세금합계,매수단가,매도단가,매수거래금액,매도거래금액,순수익
2023-09-11,2023-09-12,얼바인,2,60,44,100,200,200,400,96
2023-09-11,2023-09-21,얼바인,6,240,198,100,300,600,1800,762
2023-09-11,2023-09-24,얼바인,1,32,24,100,220,100,220,64
2023-09-11,2023-09-26,얼바인,1,35,27,100,250,100,250,88
2023-09-22,2023-09-26,얼바인,2,112,55,310,250,620,500,-287
